app/Http/Controllers/CampaignsController.php

```diff
@@ -107,13 +106,15 @@ public function showCampaign($id)
 
         return view('pages.campaign_single')
             ->with('state', [
-                'signups' => $signups,
+                'signups' => $signups->toArray()['data'],
```

Comment:
@DFurnes now `$signups` is a `Illuminate\Pagination\LengthAwarePaginator` and everything that uses `signups` expects it to be an array of signups. Probably would be better to change everything else to work with the paginator, but if we want to switch it to paginate on the front end this makes sense to me for now